web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/Plugin/views/filter/ViewsBasicFilter.php

```diff
@@ -29,9 +29,11 @@ public function query() {
     }
 
     // Parse tag filters.
-    $tag_table = $query->addTable('node__field_tags');
-    foreach ($this->value['filters']['tags'] as $tid) {
-      $query->addWhere(0, "$tag_table.field_tags_target_id", $tid);
+    if (($this->value['filters']['tags'][0])) {
```

Comment:
Checking if($var) will prevent the block of code from running but will also result in an error if the variable is not set. I suggest if(!empty($var)).

Reply:
Good catch. Fixed.